documentation/advanced_use.md

```diff
@@ -0,0 +1,8 @@
+# Advanced Use
+
+WIP
+
+Current ideas for what to include:
```

Comment:
4. Making your own cluster
5. Running on NERSC
6. How to parse/understand outputs?

Reply:
I think 4 & 5 is in use. 6 probably doesnt belong in advanced use but could be added somewhere else